cmd/atrutil/atrutil_main.go

```diff
@@ -362,6 +364,77 @@ func FindCoverage(filename string, atomicMap map[string][]*types.TestSpec) int {
 	return criteria
 }
 
+func GenerateCriteria(tid string) {
+	var atomicTests = map[string][]*types.TestSpec{} // tid -> tests
+
+	err := utils.LoadAtomicsIndexCsvPlatform(filepath.FromSlash(flagAtomicsPath), &atomicTests, flagPlatform)
+	if err != nil {
+		fmt.Println("Unable to load Indexes-CSV file for Atomics", err)
+		os.Exit(1)
+	}
+
+	if gVerbose {
+		fmt.Println("Searching for test", tid)
+	}
+
+	tests, ok := atomicTests[tid]
+	if !ok {
+		if gVerbose {
+			fmt.Println("An atomic test does not exist for this technique:", tid, "It could be an old copy of atomic-red-team repo or a fork or the criteria specifies an invalid technique")
+		}
+		// what error code should this return? for 'not found'?
+		os.Exit(1)
+	}
+
+	yaml, err := utils.LoadAtomicsTechniqueYaml(tid, flagAtomicsPath)
+
+	if err != nil {
+		fmt.Println("Could not load Yaml for ", tid)
+		os.Exit(1)
+	}
+
+	outfile, writeErr := os.OpenFile("./data/generated/"+tid+".generated.csv", os.O_CREATE|os.O_WRONLY, 0644)
+	if writeErr != nil {
+		fmt.Println("ERROR: unable to create outfile", tid, ".generated.csv", err)
+		os.Exit(2)
+	}
```

Comment:
how does the user know where the data is going?  Perhaps it's best to set `outfile=os.Stdout` and print to terminal?